tools/pvacseq/transcript_support_level_filter.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+import sys
+from lib.filter import *
+
+def define_parser():
+    parser = argparse.ArgumentParser('pvacseq transcript_support_level_filter')
+    parser.add_argument(
+        'input_file',
+        help="The all_epitopes.tsv or filtered.tsv pVACseq report file to filter"
+    )
+    parser.add_argument(
+        'output_file',
+        help="Output .tsv file containting list of of filtered epitopes based on transcript support level."
+    )
+    parser.add_argument(
+        "--maximum-transcript-support-level", type=int,
+        help="The threshold to use for filtering epitopes on the transcript support level. "
+        +"Keep all epitopes with a transcript support level <= to this cutoff. Default: 1",
+        default=1,
+        choices=[1,2,3,4,5]
+    )
+    parser.add_argument(
+        '--exclude-NAs',
```

Comment:
1) I poked around the code, but I don't see where this option is exposed to the end user (through, say, "pvacseq run").  (I probably just missed it, though)
2) It looks like the "exclude-NAs" argument applies to all filters, so there's no way to exclude NAs from filter X, but keep them in filter Y. Is that correct?  (and the desired behavior?)  I'm not sure - thinking about it...

Reply:
1. It's exposed in lib/run_argument_parser.py line 129 (https://github.com/griffithlab/pVACtools/pull/201/files#diff-a06dc88b532df643a77a13749169da4aR129). It already existed before this PR so that's why it's not showing in the diffs
2. Correct. I'm not sure if there is a good way around this. A user can always filter manually on the all_epitopes file later, if desired. This new filter is also available standalone for that purpose.

Reply:
Works for me - +1!